app/views/settings/_slack.html.erb

```diff
@@ -1,40 +1,29 @@
-<%=
-  render Primer::OpenProject::PageHeader.new do |header|
-    header.with_title { t(:label_slack_plugin) }
-    header.with_breadcrumbs([{ href: admin_index_path, text: t("label_administration") },
-                             t(:label_slack_plugin)])
-  end
-%>
-
 <div id="settings">
```

Comment:
This div can be removed as well, I'd say. Otherwise, there are two divs with id settings directly below each other